### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by MavropaliasG** [OpenRefine/OpenRefine#3017]
Hi, it would be fantastic to have a dark mode, it will make OpenRefine look more modern as well.

**Comment by thadguidry** [comment]
@MavropaliasG Yes I agree! How fast can you hack on that and submit a pull request? :-)

**Comment by Taruun** [comment]
> @MavropaliasG Yes I agree! How fast can you hack on that and submit a pull request? :-)

Can i work on that?

**Comment by MavropaliasG** [comment]
> Can i work on that?

That would be amazing. I found that https://darkreader.org/ creates a nice dark mode, can we access the CSS that it produces and just create a dark mode from that? Is it going to be easier?

**Comment by wetneb** [comment]
@Taruun thanks for volunteering, please go ahead!

**Comment by eddo888** [comment]
I too would love dark mode, is there a simple tick box on a menu we can try ? not wanting to get too into the whole CSS world at the moment.
